content/docs/06-integrations/00-kubernetes-generic.md

```diff
@@ -0,0 +1,624 @@
+<br />
+
+# Versions Supported
+
+
+<Tabs>
+
+<Tabs.TabPane tab="1.26.x" key="k8s_v1.26">
+
+## Prerequisites
+
+- A minimum of 4 CPU and 4GB Memory.
+
+
+- Users or groups mapped to a Kubernetes RBAC role.
+
+
+- Operating System (OS) dependencies as listed in the table.
+
+| OS Distribution | OS Version | Supports Kubernetes 1.26.x |
+|---------------|------------|----------------------------|
+| CentOS        | 7.7        | ✅                         |
+| Ubuntu        | 22.04      | ✅                         |
+| Ubuntu        | 20.04      | ❌                         |
+| Ubuntu        | 18.04      | ❌                         |
+
+
+## Parameters
+
+| Parameter | Description |
+|-----------|-------------|
+| ``pack.podCIDR`` | The CIDR range for Pods in cluster. This should match the networking layer property. Default: `192.168.0.0/16`|
+| ``pack.serviceClusterIpRange`` | The CIDR range for services in the cluster. This should not overlap with any IP ranges assigned to nodes or pods. Default: `10.96.0.0/12`|
+| ``kubeadmconfig.apiServer.extraArgs`` | A list of additional apiServer flags you can set.|
+| ``kubeadmconfig.apiServer.extraVolumes`` | A list of additional volumes to mount on apiServer.|
+| ``kubeadmconfig.controllerManager.extraArgs`` | A list of additional ControllerManager flags to set.|
+| ``kubeadmconfig.scheduler.extraArgs`` | A list of additional Kube scheduler flags to set.|
+| ``kubeadmconfig.kubeletExtraArgs`` | A list of kubelet arguments to set and copy to the nodes.|
+| ``kubeadmconfig.files`` | A list of additional files to copy to the nodes.|
+| ``kubeadmconfig.preKubeadmCommands`` | A list of additional commands to invoke **before** running kubeadm commands.|
+| ``kubeadmconfig.postKubeadmCommands`` | A list of additional commands to invoke **after** running kubeadm commands.|
+| ``pack.serviceDomain`` | The DNS name for the service domain in the cluster. Default: ``cluster.local``.|
+
+
+## Usage 
+  
+The Kubeadm configuration file is where you can do the following:
+
+<br />
+
+- Change the default ``podCIDR`` and ``serviceClusterIpRange`` values. CIDR IPs specified in the configuration file take precedence over other defined CIDR IPs in your environment.
+
+  As you build your cluster, check that the ``podCIDR`` value does not overlap with any hosts or with the service network and the ``serviceClusterIpRange`` value does not overlap with any IP ranges assigned to nodes or pods. For more information, refer to the [Clusters](/clusters) guide and [Cluster Deployment Errors](https://docs.spectrocloud.com/troubleshooting/cluster-deployment). 
+
+
+- Add a certificate for the Spectro Proxy pack if you want to use a reverse proxy with a Kubernetes cluster. For more information, refer to the [Spectro Proxy](/integrations/frp) guide.
+
+
+#### Configuration Changes
+
+The Kubeadm config is updated with hardening improvements, and a pod security policy has been removed.
+
+<br />
+
+#### Example Kubeadm Configuration File 
```

Comment:
@ritawatson this YAML is for PXK. The generic k8s yaml is different for each provider and very minimal compared to PXK. So you can either. Create a tab and display each versions yaml, or just make a general statement about all of them.

I personally lean more towards the later option.

Reply:
Copy/paste error - the source for the file was the PXK doc. Removed the YAML examples and added this sentence:

"#### Kubeadm Configuration File 

The default pack YAML contains minimal configurations offered by the managed provider."